cmd/dex-worker/main.go

```diff
@@ -44,6 +44,8 @@ func main() {
 	emailFrom := fs.String("email-from", "no-reply@coreos.com", "emails sent from dex will come from this address")
 	emailConfig := fs.String("email-cfg", "./static/fixtures/emailer.json", "configures emailer.")
 
+	enableRegistration := fs.Bool("enable-registration", true, "Allows users to self-register")
```

Comment:
Should this default to true? While we might break existing installs if it defaults to false, it seems safer to have open registration be the explicit choice.